tests/python/gaia-ui-tests/gaiatest/tests/functional/system/test_system_notification_bar.py

```diff
@@ -29,9 +29,12 @@ def test_notification_bar(self):
 
         utility_tray.wait_for_notification_container_displayed()
 
+        # Assert there is one system notification is listed in notifications-container
         notifications = utility_tray.get_notifications(for_app='system')
-        self.assertEqual(1, len(notifications), 'Expected one system notification.')
-        email = notifications[0].tap_notification()
+        self.assertEqual(1, len(notifications), 'Expected one notification.')
```

Comment:
Why do we change the error message? It seems like we still wait for a system notification to be displayed.